.github/workflows/release.yml

```diff
@@ -42,7 +42,9 @@ jobs:
         uses: peaceiris/actions-gh-pages@v3
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
-          publish_dir: doc/output
+          publish_dir: |
+                        doc/output
+                        !doc/output/.git
```

Comment:
Are you sure you want to do this here?

Reply:
I was looking for a way to test that line but I didn't know-how. Is this not the same as build-and-test.yml?